packages/arco-lib/src/hooks/useStateValue.ts

```diff
@@ -22,7 +23,7 @@ export const useStateValue = <
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  useEffect(() => {
```

Comment:
Is there any other components using this hook?

Reply:
All components that have the `updateWhenDefaultValueChanges` property will use this hook

Reply:
Then it is dangerous to modify it. It could cause unexpected problems. A new hook should be created.

Reply:
> Then it is dangerous to modify it. It could cause unexpected problems. A new hook should be created.

ok，perhaps it would be more appropriate to handle it in the collapse component